app/views/landing_page/_meta.erb

```diff
@@ -61,3 +61,6 @@
   <meta name="twitter:description" content="<%= p["description"]["value"] %>" />
 <% end %>
 
+<% if p["google_site_verification"] %>
+  <meta name="google_site_verification" content="<%= p["google_site_verification"]["value"] %>" />
```

Comment:
I'm not entirely sure, but I think this might cause "undefined" problems, and therefore it would be better to have this line instead:
```  <meta name="google_site_verification" content="<%= p.dig("google_site_verification", "value") %>" />```

ping @rap1ds and @ovan